testapp/views.py

```diff
@@ -67,14 +67,14 @@ def index(request):
 
 def info(request):
   print("RESTAURANTS", request.POST)
-  city = request.POST['location']
+  city = request.POST['location'].title()
```

Comment:
Why do we call `.title()` here?

Reply:
ahh that's just to make the first letter capital (like boston --> Boston)